package/thingino-kopt/thingino-kopt.mk

```diff
@@ -316,6 +316,14 @@ define THINGINO_KOPT_LINUX_CONFIG_FIXUPS_NETFILTER
 endef
 endif
 
+# TUN required for zerotier#
+ifeq ($(BR2_PACKAGE_ZEROTIER_ONE),y)
```

Comment:
Do you expect BR2_PACKAGE_ZEROTIER_ONE=y in local.fragment? Could yo clarify how to enable zerotier?